### PR TITLE
Limit batch size

### PR DESCRIPTION
**Pull request by hinthornw**

### CODE REVIEW

File: python/langsmith/client.py [hunk 1]
```diff
@@ -2159,24 +2159,47 @@ def get_test_results(
...
+            examples = self.list_examples(example_ids=batch)
+            return [
                 {
+                    "example_id": example.id,
+                    **{f"reference.{k}": v for k, v in (example.outputs or {}).items()},
+                }
+                for example in examples
+            ]
+
+        batch_size = 50
```
Comment (hinthornw) [inline python/langsmith/client.py:2194]: Main thing is this - bump down to 50 from 100 (was causing errors)

File: python/pyproject.toml [hunk 2]
```diff
@@ -53,6 +53,7 @@ vcrpy = "^6.0.1"
 fastapi = "^0.110.1"
 uvicorn = "^0.29.0"
 pytest-rerunfailures = "^14.0"
+rapidfuzz = "^3.9.0"
```
Comment (nfcampos) [inline python/pyproject.toml:56]: this here on purpose?

Comment (hinthornw) [reply python/pyproject.toml:56]: was for linting warnings locally

Comment (hinthornw) [reply python/pyproject.toml:56]: just a dev dep but i can remove

**Review by langchain-infra — APPROVED** [review]

**Pull request merged**